framework/basehandlers.py

```diff
@@ -552,6 +552,7 @@ def FlaskApplication(import_name, routes, post_routes, pattern_base='', debug=Fa
   """Make a Flask app and add routes and handlers that work like webapp2."""
 
   app = flask.Flask(import_name,
+    static_folder=settings.get_static_path(),
```

Comment:
Doesn't it default to 'static'?

Reply:
good catch! fixed. 

Also, as a result of you bringing this up, I did some more digging for the existing template_folder parameter I added earlier. I added static_folder here because I did it for template_folder. It turns out, I only need  to set the template_folder for the unit tests. I will be removing it here in #2323

Reply:
Correction, I will be doing this in the PR that removes django completely.